rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -242,8 +250,51 @@ Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to
 
 Once it's ready, transition the project to the next phase of its life-cycle. You can always transition back to change the project's design or implementation.
 
+In the **Design** phase, you can do the following:
+
+* Create or change a process model in RPA Manager.
++
+Already implemented activities don’t lose their implementation if you don’t delete them. 
+* Transfer the automation containing the new or changed models to the Build Phase.
+
+In the **Build** phase, in RPA Builder, you can do the following:
+
+* Download the model from RPA Manager.
+* Change the model.
+* Implement the activities.
+* If you want to version workflows, create and reuse activities in the Activity Library.
+* Upload the changed and implemented model to the repository in RPA Manager. 
++
+There is always only one version of a process in the repository in RPA Manager. There might be another local version in RPA Builder. If you accidentally deleted an already implemented Activity, you might still be able to find it in RPA Builder - as long as you do not overwrite the model by downloading.
```

Comment:
```suggestion
Only one version of a process exists in the repository in RPA Manager. If you accidentally delete an already implemented activity, a local version might be available in RPA Builder — as long as you don't overwrite the model by downloading.
```

Reply:
What does this mean: "don't overwrite the model by downloading"?

Does the local version exist in RPA builder until they download it?